packages/system-tests/src/areas/common.ts

```diff
@@ -71,7 +71,7 @@ export class CommonActions {
   }
 
   public async getConsoleOutput(): Promise<any> {
-    const htmlTag = `div[class="view-lines"]`;
+    const htmlTag = `div[id="workbench.panel.output"] div[class="view-lines"]`;
```

Comment:
VSCode has 2 different areas which use `view-lines` as a class; the monaco editor & the terminal output panel. This change makes sure we always look for the content in the terminal's output panel.